main.js

```diff
@@ -56,12 +58,23 @@ ideaSection.addEventListener('click', function(event) {
     if (ideas[i].id === deleteIdea) {
       ideas.splice(i, 1)
     }
-   } renderIdea(ideas);
+   }
+   renderIdea(ideas);
+  //  favoriteCards(event);
+})
 
-   var favorite = event.target.parentNode.children[0]
-   var favoriteActive = event.target.parentNode.children[1]
-   favorite.classList.toggle('hidden')
-   favoriteActive.classList.toggle('hidden')
-   console.log('favorite: ', favorite.classList)
-   console.log('favorite active: ', favoriteActive.classList)
-})
\ No newline at end of file
+// function favoriteCards(event) {
+//   event.preventDefault()
+//   if(event.target.classList.contains('star')){
+//     event.target.src="./Assets/star-active.svg"
```

Comment:
This line 69 is somehow invoked before clicking on the star. Also, when the image src is updated to the active star, it doesn't update on the DOM.